packages/client/src/App.js

```diff
@@ -17,6 +18,7 @@ function App() {
           {/* <Route path="/" element={<LandingPage />} /> */}
           <Route path="/CreateNewTeamPage" element={<CreateTeam />} />
           <Route path="*" element={<PageNotFound />} />
+          <Route path="/PastRetroPage" element={<PastRetroPage />} />
```

Comment:
There's nothing inherently wrong with naming routes like this.
Instead of naming routes based solely on the component they render, consider naming them based on the purpose or content of the page. For example, if the page displays past retro sessions, you might name it something like `/retros/past` or `/retros/new`. This makes it clearer what content the route leads to.

Reply:
Renamed the route as suggested. Thank you Dan